queries/julia/injections.scm

```diff
@@ -1,8 +1,13 @@
-; TODO: re-add when markdown is added.
-; ((triple_string) @markdown
-;   (#offset! @markdown 0 3 0 -3))
+((string_literal) @markdown
+ (#match? @markdown "^\"\"\"")
+ (#offset! @markdown 0 3 0 -3))
```

Comment:
This is unrelated to regexes

Reply:
True, but I noticed the todo and decided to fix it. I'll split the commit.

Reply:
It should probably work fine now